tests/docs/smoke-all/2023/01/16/md-captions.qmd

```diff
@@ -0,0 +1,46 @@
+---
+title: markdown in captions
+format:
+  html: default
+  latex: default
```

Comment:
Did you check to see that our test suite supports checking on multiple files at the same time?

What you did here is totally reasonable, and should ideally work. But I don't think we have documents in smoke-all that render to multiple formats, and then test different things on different formats. I suspect this one is only getting tested in HTML.

Reply:
Yes!